contracts/Lens/JoeLensV2.sol

```diff
@@ -0,0 +1,364 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.5.16;
+pragma experimental ABIEncoderV2;
+
+import "../JErc20.sol";
+import "../Joetroller.sol";
+import "../JToken.sol";
+import "../PriceOracle/PriceOracle.sol";
+import "../EIP20Interface.sol";
+import "../Exponential.sol";
+import "../IRewardDistributor.sol";
+
+/**
+ * @notice This is a version of JoeLens that contains write transactions
+ * and pulls reward speeds from a RewardDistributor.
+ * @dev Call these functions as dry-run transactions for the frontend.
+ */
+contract JoeLensV2 is Exponential {
+    /// @notice Administrator for this contract
+    address public admin;
+
+    /// @notice The module that handles reward distribution
+    address payable public rewardDistributor;
+
+    /// @notice The native token symbol for this contract
+    string public nativeSymbol;
+
+    /// @notice Metadata for a market
```

Comment:
I noticed we use `///` over `/** */` natspec format for multiline comments on structs in other contracts, don't know if we'd rather just use `/** */` for all multilines to keep it consistent

Reply:
Either is fine. Openzeppelin for example, likes to us `/** */`